### PR TITLE
Set a feature flag for access codes

### PR DESCRIPTION
**Pull request by ralren**
## Description

Set up a server-side feature flag for the new access code feature.

## Setup

Add `export REQUIRES_ACCESS_CODE = false` to your `.envrc.local` file. Change to true to test if the enter access code page shows up.

```sh
make server_run
make client_run
```

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/tree/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/tree/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/tree/master/docs/database.md#zero-downtime-migrations))
  * [ ] Have been communicated to #dp3-engineering
  * [ ] Secure migrations have been tested using `scripts/run-prod-migrations`
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* Any new client dependencies (Google Analytics, hosted libraries, CDNs, etc) have been:
  * [ ] Communicated to @ntwyman
  * [ ] Added to the list of [network dependencies](https://github.com/transcom/mymove#client-network-dependencies)
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/166453611)

### CODE REVIEW

File: pkg/handlers/internalapi/service_members.go [hunk 1]
```diff
@@ -124,6 +126,7 @@ func (h CreateServiceMemberHandler) Handle(params servicememberop.CreateServiceM
 		BackupMailingAddress:   backupMailingAddress,
 		SocialSecurityNumber:   ssn,
 		DutyStation:            station,
+		RequiresAccessCode:     true,
```
Comment (chrisrcoles) [inline pkg/handlers/internalapi/service_members.go]: why is this `true`, shouldn't we pass `requiresAccessCode` here?

**Comment by chrisrcoles** [comment]
Hey @ralren - great work. This looks awesome! Can we add a test that checks when `env` variables are set to get the expected behavior?

File: pkg/handlers/internalapi/users_test.go [hunk 2]
```diff
@@ -60,11 +61,15 @@ func (suite *HandlerSuite) TestServiceMemberLoggedInUserRequiringAccessCodeHandl
 		mock.AnythingOfType("UUID"),
 	).Return(&accessCode, nil)
 
-	handler := ShowLoggedInUserHandler{handlers.NewHandlerContext(suite.DB(), suite.TestLogger()), accessCodeFetcher}
+	context := handlers.NewHandlerContext(suite.DB(), suite.TestLogger())
```
Comment (ralren) [inline pkg/handlers/internalapi/users_test.go]: @chrisrcoles, is that what you were thinking of for testing `env`?

Comment (chrisrcoles) [reply pkg/handlers/internalapi/users_test.go]: yes!

**Comment by chrisrcoles** [comment]
LGTM - we are waiting on product approval to merge this.

**Pull request closed without merging**